aulia nada safira_211011401270/tugas2.docx

```diff
@@ -0,0 +1,21 @@
+**sistem order direstgioran**
+1. fungsional
+- Login atau masuk sebagai tamu
+- Pelanggan menelusuri menu, memilih item, dan menambahkannya ke keranjang
+- Konfirmasi pesanan dan memilih metode pembayaran
+- Staf dapur memproses dan menyiapkan makanan
+- Staf mengantar makanan ke meja pelanggan
+2. non-fungsional 
+- Semua data pengguna dienkripsi saat disimpan dan saat transmisi
+- Mekanisme caching digunakan untuk mengurangi waktu akses ke data yang sering digunakan
+- Backup data dilakukan secara otomatis setiap 30 menit
+- Desain antarmuka diuji dengan berbagai kelompok pengguna untuk memastikan kemudahan penggunaan
+- Pengujian kompatibilitas dilakukan pada berbagai model tablet dengan sistem operasi iOS dan Android
+3. sistem untuk koki
+- Daftar pesanan yang diterima dengan rincian item dan catatan khusus
+- Informasi lengkap tentang setiap item dalam pesanan, termasuk bahan, ukuran porsi, dan instruksi khusus
+- Mengirim notifikasi ke pelayan atau sistem layanan ketika pesanan siap untuk diantarkan
+- Melacak ketersediaan bahan baku yang dibutuhkan untuk setiap pesanan
+- Menyediakan resep lengkap untuk setiap menu, termasuk langkah-langkah persiapan dan waktu memasak
+- Menyediakan resep lengkap untuk setiap menu, termasuk langkah-langkah persiapan dan waktu memasak.
+
```

Comment:
Overall, you did a good job. Hanya beberapa catatan saja yaa. Dan akan lebih rapi kalau disajikan di dalam tabel. Bisa dipisah usernya apa saja seperti yang sudah dicontohkan, sistem order dan sistem koki juga dipisahkan ke dalam tabel sehingga lebih rapi dan lebih tertata antara kebutuhan fungsional dan non-fungsionalnya.